packages/safe-apps-sdk/package.json

```diff
@@ -37,9 +36,7 @@
     "prettier": "^2.2.1",
     "rimraf": "^3.0.2"
   },
-  "peerDependencies": {
-    "web3-core": "^1.3.0"
-  },
+  "peerDependencies": {},
```

Comment:
you can exclude "peerDependencies" at all

Reply:
I used `yarn remove` and that one didn't remove it, so I didn't touch it further